steering/CHARTER.md

```diff
@@ -81,7 +81,11 @@ and community, beginning in August 2020.
         the seat for the term.
     1. Because the goal of Community Seats is to increase the perspectives on
     the Steering Committee, employees of a Company that holds Contribution
-    Seats are ineligible to be elected to hold a Community Seat.
+    Seats are ineligible to hold a Community Seat.
+        1. If a Community Seat thus becomes vacant due to the holding company
+        earning a Contribution Seat, a special election will be held in the
+        weeks following these events, in the same manner as described above
+        for the regular Community Seat elections.
```

Comment:
```suggestion
        1. If a Community Seat becomes vacant due to a Company earning a
        Contribution Seat while holding a Community Seat, a special election
        will be held, in the manner descried above for the annual Community
        Seat elections.
```

PTAL, tried to tidy the language a little.

Now we're making special elections a thing, should we also apply the same process to "If an individual changes company affiliation mid-term"?